bundles/paikkatietoikkuna/tiles3d/domain/Tiles3DLayer.js

```diff
@@ -14,7 +14,25 @@ Oskari.clazz.define('Oskari.map3dtiles.bundle.tiles3d.domain.Tiles3DLayer',
         /* Layer Type */
         this._layerType = 'tiles3d';
     }, {
-
+        /**
+         * @method getStyleDef
+         * @param {String} styleName
+         * @return {Object}
+         */
+        getStyleDef: function (styleName) {
+            if (this._options.styles) {
```

Comment:
Add a check for options to be present in mapLayerJson before referencing styles.